CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+## Getting Started
+
+First, you'll need a couple things installed: Docker and asdf.
+With those, you can install all the other tooling that is necessary.
+Assuming you're on macOS, run the following.
+
+```
+brew install coreutils openssl readline xz
+brew install docker asdf
+asdf plugin-add elixir
+asdf plugin-add nodejs
+asdf install
+```
+
+Optionally, you may wish to install tmuxp, a Python package, to get everything started via tmux.
+```
+brew install tmux
+/usr/local/opt/asdf/plugins/nodejs/bin/import-release-team-keyring
+asdf plugin-add python
+CFLAGS="-I$(brew --prefix openssl)/include" LDFLAGS="-L$(brew --prefix openssl)/lib" asdf install
+pip install tmuxp
+```
+
+If you have installed tmuxp, run `tmuxp load ./tmuxp.yaml`.
+Otherwise, you should still read through tmuxp.yaml to see the commands necessary to get all the components running.
+Once everything is up, you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+
+## Branching
+
+Before making any changes, please make a branch.
+Each branch should be for a single change and should have no more than one or two commits.
+When in doubt, err on the side of fewer commits.
```

Comment:
Why fewer commits? Both for my curiosity and also for people reading this.

Reply:
I'll answer here and bring this back into the doc after.

Issues should be small (issue template to follow: #18) so the commits should be, too. That in itself should mitigate the need for multiple commits. Furthermore, commits should be complete, atomic, and easily reverted. If there are 10 commits for adding a form, it would be a pain to revert it if something went wrong.